### PR TITLE
Update login.js

### PR DESCRIPTION
**Pull request by DeepaHiras**
Verified the current date selected

### CODE REVIEW

File: cypress/integration/login.js [hunk 1]
```diff
@@ -21,4 +21,11 @@ describe('Login functionality', function() {
         cy.get('.user-info__title').contains('TestCon User 9')
         cy.get('.main-nav').find('li').should('have.length', 2)
     })
+
+    it('Validate what date is selected as today', function() {
+
+        cy.get('.calendar--selected').contains('13')
```
Comment (MonkeviciuteGerda) [inline cypress/integration/login.js:27]: Will it work tomorrow? Does this selector only finds one element?

**Review by MonkeviciuteGerda — CHANGES_REQUESTED** [review]

File: cypress/integration/login.js (hunk 1 above)
Comment (DeepaHiras) [reply cypress/integration/login.js]: Thank you. Have made the changes

File: cypress/integration/login.js [hunk 2]
```diff
@@ -21,4 +21,10 @@ describe('Login functionality', function() {
         cy.get('.user-info__title').contains('TestCon User 9')
         cy.get('.main-nav').find('li').should('have.length', 2)
     })
+
+    it('Validate what date is selected as today', function() {
+
+        cy.get('.calendar--selected').should('have.class', 'calendar--today')
```
Comment (MonkeviciuteGerda) [inline cypress/integration/login.js:27]: You should validate the actual date, like today is 13th, tomorrow it will be 14th.

**Review by MonkeviciuteGerda — CHANGES_REQUESTED** [review]

File: cypress/integration/login.js (hunk 2 above)
Comment (DeepaHiras) [reply cypress/integration/login.js:27]: Fixed

File: cypress/integration/login.js [hunk 3]
```diff
@@ -21,4 +21,12 @@ describe('Login functionality', function() {
         cy.get('.user-info__title').contains('TestCon User 9')
         cy.get('.main-nav').find('li').should('have.length', 2)
     })
+
+    it('Validate what date is selected as today', function() {
+
+        const date = new Date();
+        const currentDate = date.getDate()
+        cy.get('.tile__title').contains(currentDate)
```
Comment (MonkeviciuteGerda) [inline cypress/integration/login.js:29]: you should assert the selected day in calendar, not the side of the screen.

**Review by MonkeviciuteGerda — CHANGES_REQUESTED** [review]

File: cypress/integration/login.js (hunk 3 above)
Comment (DeepaHiras) [reply cypress/integration/login.js]: Done

**Review by MonkeviciuteGerda — APPROVED** [review]
Well done!

**Pull request merged**